### PR TITLE
Add index.html

### PR DESCRIPTION
**Pull request by barnabaspakay**

### CODE REVIEW

**Comment by Atesz99** [comment]
nice

**Comment by bmarci888** [comment]
:D

**Comment by GregPappDev** [comment]
Good Job!

**Comment by Peterfug** [comment]
Nic

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>Hello</p>
```
Comment (barnabaspakay) [inline index.html:10]: Hello? rly?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (GregPappDev) [inline index.html:7]: title ok

**Pull request merged**